src/workbookManager.ts

```diff
@@ -67,26 +128,30 @@ export class WorkbookManager {
         }
         const parser: DOMParser = new DOMParser();
         const serializer = new XMLSerializer();
-
+        const refreshOnLoadValue = refreshOnOpen ? "1" : "0";
         const connectionsDoc: Document = parser.parseFromString(
             connectionsXmlString,
             "text/xml"
         );
         let connectionId = "-1";
```

Comment:
might be a cleaner solution than what I've introduced originally - leave it undefined and if you reach current check - that means we're good to continue (since we throw otherwise)